package.json

```diff
@@ -19,16 +19,16 @@
     "name": "TileDB",
     "email": "support@tiledb.com"
   },
-  "homepage": "https://github.com/TileDB-Inc/PyBabylonJS",
+  "homepage": "https://github.com/TileDB-Inc/TileDB-PyBabylonJS",
   "bugs": {
-    "url": "https://github.com/TileDB-Inc/PyBabylonJS/issues"
+    "url": "https://.com/TileDB-Inc/TileDB-PyBabylonJS/issues"
```

Comment:
Thanks for the change! if you can add `github.com` back to the url, we'll get this merged!

Reply:
Thanks, and nice talking to your team!